### DISCUSSION TITLE
Lazy Hydration/Lighthouse 6 Performance

### DISCUSSION THREAD

**Discussion by lilbumblebear** [react-static/react-static#1488] [General]
Hi, has anyone else tested their react-static site performance using Lighthouse 6?  We are seeing really bad performance scores, and are trying to get improvements.  

A lot of the time seems to be spent on "Script Evaluation"

One of the things we have seen from other react frameworks was the concept of lazy hydration [reference React Storefront](https://www.moovweb.com/post/lighthouse6-website-frontend-optimization)  

Has anyone tried using something like [react-lazy-hydration](https://github.com/hadeeb/react-lazy-hydration) with react-static? Curious if it works or if you have seen improvements in performance for SEO?

**Comment by SleeplessByte** [comment]
Using lazy hydration should "Just Work". It won't solve the issue here tho. You're bundle is probably quite big. We should really "first class" support lazy routes/components, which would solve most of your issues.

For now, I would try to optimise the bundle size. We're still hitting 90+ range with Lighthouse 6 without any changes.
